pkg/operators/klusterlet/controllers/ssarcontroller/klusterlet_ssar_controller.go

```diff
@@ -123,10 +123,7 @@ func (c *ssarController) sync(ctx context.Context, controllerContext factory.Syn
 		defer c.deleteSSARChecking(klusterletName)
 
 		klog.V(4).Infof("Reconciling Klusterlet %q", klusterletName)
-		klusterletNS := klusterlet.Spec.Namespace
-		if klusterletNS == "" {
-			klusterletNS = klusterletNamespace
-		}
+		klusterletNS := helpers.KlusterletNamespace(klusterlet)
```

Comment:
I think we can delete the const `klusterletNamespace` since it will not be used.

Reply:
sure. 
@qiujian16  could we put the const vars (like the condition names, secret names, namesapce names etc.)  of the packages into a specified file?  some const may be defined repeatedly if we use them in multi packages.

Reply:
yes, I think we should maintain the condition types in APIs. secret/namespace depends. A namespace defined in API may also should be defined as a const in API.